### DISCUSSION TITLE
Que ha pasado con el Live?

### DISCUSSION THREAD

**Discussion by maikcore** [LaunchX-InnovaccionVirtual/CursoIntroPython#160] [General]
Buenas, es la primera vez que entro a un live a tiempo pero no veo señal, a qué hora empiezan normalmente?

**Comment by maikcore** [comment]
ya :p

**Comment by MSuarez0** [comment]
ya comenzo, suerte :D

**Comment by Dfredude** [comment]
Ya inicio!
![image](https://user-images.githubusercontent.com/72330887/153692761-a3af6cc5-1124-460b-a3df-44f4aa19765a.png)
